### PR TITLE
USHIFT-3455: Add apigroup to skip a FeatureGate test in MicroShift

### PR DESCRIPTION
**Pull request by pacevedom**

### CODE REVIEW

File: test/extended/operators/server_side_apply_examples.go [hunk 1]
```diff
@@ -90,7 +90,7 @@ var _ = g.Describe("[sig-apimachinery]", func() {
 			}
 		})
 
-		g.It("should clear fields when they are no longer being applied in FeatureGates", func() {
+		g.It("should clear fields when they are no longer being applied in FeatureGates [apigroup:config.openshift.io]", func() {
```
Comment (dgoodwin) [inline test/extended/operators/server_side_apply_examples.go:93]: This looks good, @stbenjam is there a default stablemap impl in https://github.com/openshift-eng/ci-test-mapping that will automatically omit the apigroup tag from the stable test name? Or will this require a custom rename?

Comment (stbenjam) [reply test/extended/operators/server_side_apply_examples.go:93]: I don't think so but we should just handle stuff like this https://issues.redhat.com/browse/TRT-1716

**Pull request merged**